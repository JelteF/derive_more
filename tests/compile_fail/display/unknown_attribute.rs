#[derive(derive_more::Display)]
#[display(fmt = "Stuff({})", bar)]
#[display(unknown = "unknown")]
pub struct Foo {
    bar: String,
}

fn main() {}
