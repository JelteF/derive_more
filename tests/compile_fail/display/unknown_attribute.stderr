error: Unknown `unknown` attribute argument. Allowed arguments are: fmt, bound
 --> tests/compile_fail/display/unknown_attribute.rs:3:11
  |
3 | #[display(unknown = "unknown")]
  |           ^^^^^^^
